content/index.md

```diff
@@ -1,43 +1,70 @@
 type: page
 slug: index
 url: index.html
-title: Home
+title: PyCon UK 2016
 callout_big_1: PyCon UK is your community-led
 callout_big_2: volunteer-run conference
 callout_small: Cardiff City Hall from Thursday 15th to Monday 19th September 2016
 ---
 
 PyCon UK is the annual gathering of the UK Python community and its friends
 from around the world.  This year, the conference is held at [Cardiff City
-Hall](http://www.cardiffcityhall.com/), from Friday 16th to Monday 19th
-September, with a day of introductory workshops on Thursday 15th.
+Hall](http://www.cardiffcityhall.com/), from Thursday 15th to Monday 19th
+September.
+
+## Overview of the programme
```

Comment:
`h2`s are too big for these headings.  I think `h3`s are a bit too big, but `h4`s are much too small.  Could you change them to `h3`s?

I like breaking things up with headings though.